docs/change-log/index.md

```diff
@@ -14,6 +16,20 @@
 - [Limitations](#limitations)
 - [Deprecations](#deprecations)
 
+## v1.3.1
+
+### Closed issues
+
+| ID   | Description                                                    |
+| --- | ----------------------------------------------------------------- |
+| ML-3764 | The tutorial 02-model-training requires scikit-learn 1.2. (Previously pointed to 1.0.) [View in Git](https://github.com/mlrun/mlrun/pull/3437).  |
+| ML-3823 | Projects that have very large yaml (configuration) files does not cause system crash when previewing in UI. [View in Git](https://github.com/mlrun/mlrun/pull/3560).  |
```

Comment:
```suggestion
| ML-3823 | Reducing larged-size projects effecting with backend slowness [View in Git](https://github.com/mlrun/mlrun/pull/3560). 
```

in addition to 1.3.1:

- Optimized /projects endpoint to work faster
- Reduce too verbose logs on backend side

Reply:
@liranbg - pls respond with full comments and JIRA #s! I can't see anything in 1.3.1 related to "Optimized /projects endpoint to work faster"
Unless you didn't mean that these are additional fixes in 1.3.1?